packages/grafana-ui/src/components/Collapse/Collapse.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Preview, Props } from '@storybook/addon-docs/blocks';
+import {Collapse} from "./Collapse";
+
+# Collapse
+
+A component that allows users to toggle the display of sections of content.
```

Comment:
I don't know for sure how it works because it's not in the storybook yet, but does it work like an accordion?
I would probably say "collapse and expand sections of content" instead of "toggle the display", which is a bit unclear what exactly it means. If it's an accordion we could explain in even more detail how it works – which part is clickable/where does the toggling happen? Is there a part that doesn't get collapsed? How are the sections separated visually? Describing in more detail what exactly the component does and how it looks might go a long way :)

Reply:
Yeah, it's a basic accordion. Not sure we'd be that detailed with component descriptions especially when they are quite well known components. But I agree that the description could use some tweaking.